app/views/editor/ai-document/AIDocumentSearchView.js

```
@@ -14,13 +14,19 @@ module.exports = (AIDocumentSearchView = (function () {
   AIDocumentSearchView = class AIDocumentSearchView extends SearchView {
     static initClass () {
       this.prototype.id = 'editor-ai-document-home-view'
-      this.prototype.modelLabel = 'Document'
+      this.prototype.modelLabel = 'AI Document'
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)